muxers/yamux/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Unreleased
+
+- Implement `From<io::Error> for YamuxError` instead of `Into`.
+  [PR 2169](https://github.com/libp2p/rust-libp2p/pull/2169)
+
 # 0.34.0 [unreleased]
 
 - Update dependencies.
```

Comment:
```suggestion
# 0.34.0 [unreleased]

- Update dependencies.

- Implement `From<io::Error> for YamuxError` instead of `Into`.
  [PR 2169](https://github.com/libp2p/rust-libp2p/pull/2169)
```